commands/peers/peer-rpc-svc.go

```diff
@@ -31,7 +33,7 @@ var (
 	etcdConfFile = etcdConfDir + "etcdenv.conf"
 )
 
-// ValidateAdd() will checks all validation for AddPeer at server side
+// ValidateAdd will checks all validation for AddPeer at server side
```

Comment:
The grammar here sucks. "...will checks.." !.
Could you please reword this and the stuff below, along the lines of `Validate* validates ....`.